conf/openlibrary.yml

```diff
@@ -194,6 +194,9 @@ plugin_recaptcha:
     public_key: ""
     private_key: ""
 
+plugin_inside:
+    search_endpoint: "https://api.archivelab.org/search/openlibrary"
```

Comment:
@gdamdam says we should use the https://be-api.us.archive.org/fts/v1/search API for developers (its likely to have better uptime -- I'm likely going to take the archive labs API down in favor of the be-api endpoint)